src/contract.rs

```diff
@@ -138,6 +140,33 @@ impl<T: Transport> Instance<T> {
     }
 }
 
+/// `Deployments` implementation for an `Instance`. This type is not intended to
+/// be used directly but rather through the `Instance::deployed` API.
+#[derive(Debug, Clone)]
+pub struct Networks {
```

Comment:
I find this name a bit confusing. Why would a network have a field called ABI? Maybe Deployments is a better name?

Reply:
Definitely a better name!